samples/dictionary.sample.yml

```diff
@@ -0,0 +1,763 @@
+# Key to Entry Fields
+#
+# Each object represents a *dictionary entry*.
+#
+# category:                                         # The lexical category for the entry. This is displayed to the end user as-is, but it is also used by the dictionary app for ...? For Plains Cree, the value of this field is whatever is in Arok's `\ps` field in the Toolbox file. For now, each entry may only have one lexical category.
+# FST:                                              # Object containing information for the FST.
+#   lemma:                                          # The lemma used by the FST, in the proto-orthography.
+#   stem:                                           # The stem used by the FST, in the proto-orthography.
```

Comment:
Strictly speaking, there can be three types of "stems".
1. The "stem" that the lexicographer has specified. In CW, this is "stem".
2. The "stem" that iDict shows to a user. Maybe this should be "iDictStem". 
3. The "stem" that is used as the basis for the LEXC stem. Maybe this should be "fststem" or "lexcstem".

An example for the entry" ay-itêw" of all three types are the following:
1. "ay"
2.  "ay-it-"
3. "ay-it3"

In the case of crk, 2 and 3 are based on some logic operating on a combination of a. the stem in the original dictionary source, b. the (lexical) category, and b. the lemma (head). Whatever these variants are, they should be created on the crk-db side.

iDict does not need any of these three variants of stems for anything other than showing it. For now, we've been opting to show option 2. Also, for dictionary purposes, we can have an separate field known as "stem", which can be kept independent of whatever the LEXC code needs.

FInally, we know that there can be entries with more than one stem, for crk e.g. atim -> atimw- (Non-possessed); têm- (Possessed).

Reply:
Good point - those uses of "stem" are helpful to distinguish.

So currently we're using the FST stem as the iDictStem displayed to the user. Is this correct?